topics/metagenomics/tutorials/nanopore-16S-metagenomics/tutorial.md

```diff
@@ -0,0 +1,355 @@
+
+![FasTQC plots CG](../../images/metagenomics-nanopore/fastqc_per_sequence_gc_content_plot.png "Per sequence GC content")
+
+Let's have a look at figure 4, which shows the content of the GC by sequence.
+
+>
+> ### {% icon question %} Questions
+>
+> What can cause GC content to show a bimodal peak?
+>
+> > ### {% icon solution %} Solution
+> >
+> > There are several possible causes for the presence of twin peaks in the GC content. One possible explanation is the presence of adapters in the sequence. Another possible cause is some kind of contamination, such as chimeras.
+> >
+> {: .solution}
+>
+{: .question}
+
+> ### {% icon comment %} Comments
+> For more information on the topic of quality control, please see our training materials [here](https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/quality-control/tutorial.html).
+{: .comment}
+
+
+# Improve the dataset quality
+
+In order to improve the quality of our data, we will use two tools presented above, __porechop__ and __fastp__.
+
+## Adapter and chimera removal with porechop
+
+Nanopore sequencing technology requires to ligate adapters to both ends of genomic material to facilitate the strand capture and loading of a processive enzyme at the 5'end, boosting the effectiveness of the sequencing process. Adapter sequences should be removed because they __can interfere with aligment of reads__ to 16S rRNA gene reference database, for which we will use the __porechop__ tool ({% cite rrwick2017 %}).
+
+> ### {% icon comment %} Comment
+> If you are interested in Nanopore sequecing technology, you can find more information in {% cite Jain2016 %}.
+> 
+{: .comment}
+
+
+On the other hand, __chimeric sequences__ are considered a contaminant and should be removed because they can result in artificial inflation of the microbial diversity. With __Porechop__ you can eliminate them. 
+
+> ### {% icon hands_on %} Hands-on: Remove adapters with porechop
+>
+> 1. {% tool [Porechop](toolshed.g2.bx.psu.edu/repos/iuc/porechop/porechop/0.2.3) %} with the following parameters:
+>    - *"Input FASTA/FASTQ"*: {% icon param-collection %} `soil collection`
+>    - *"Output format for the reads"*: `fastq`
+> 2. Rename the output as `soil collection trimmed`
+>
+{: .hands_on}
+
+## Filter sequences with fastp
+
+To increase the specificity of the analysis, we will select the reads with lengths between 1000 bp and 2000 bp, which are more informative from a taxonomic point of view, because they include both preserved and hypervariable regions of the 16S rRNA gene. In addition, sequences will be filtered on a minimum average read quality score of 9, according to the recommendations from {% cite Nygaard2020 %}. This stage will be carried out through the use of __fastp__ ({% cite Chen_2018 %}), an open-source tool designed to process FASTQ files.
```

Comment:
You're also filtering just because you would expect the sequenced 16s rRNA gene has that length right. So you can assume that sequences shorter or longer are wrong in the first place?

```suggestion
To increase the specificity of the analysis, we will select the reads with lengths between 1000 bp and 2000 bp, which are more informative from a taxonomic point of view, because they include both preserved and hypervariable regions of the 16S rRNA gene. In addition, sequences will be filtered on a minimum average read quality score of 9, according to the recommendations from {% cite Nygaard2020 %}. This stage will be carried out through the use of {% tool [fastp](toolshed.g2.bx.psu.edu/repos/iuc/fastp/fastp/0.20.1+galaxy0) %} ({% cite Chen_2018 %}), an open-source tool designed to process FASTQ files.
```